### DISCUSSION TITLE
Monitor not detected if using deck (Targus)

### DISCUSSION THREAD

**Discussion by cvivojim** [emoacht/Monitorian#394] [Compatibility]
Second monitor is not detected by the app when I use the deck (Targus) to connect it through USB C port

**Comment by emoacht** [comment]
Thanks for reporting.
I will move this issue to the discussion as Incompatibility Information.

**Reply by cvivojim** [reply]
So no fix will be expected right?

**Reply by emoacht** [reply]
Unfortunately no. It must be that deck's issue.
